gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -958,11 +958,11 @@ gst_tensor_filter_compare_tensors (GstTensorsInfo * info1,
     }
 
     line =
-        g_strdup_printf ("%2d : %s | %s %s\n", i, left, right,
-        g_str_equal (left, right) ? "" : "FAILED");
-    if (left && left[0] != '\0')
+        g_strdup_printf ("%2d : %s | %s %s\n", i, GST_STR_NULL (left),
```

Comment:
left and right values cannot be null string.

we need to consider line 930 ```gchar null[] = "";```
remove this and add code to check null string.
2.

Reply:
What about 'g_strcmp0()` instead of `g_str_equal()`?
Well, NULL is passed to g_strcmp0(), it works without segmentation faults.